backend/controllers/appointments/prepostAppoinments.js

```diff
@@ -292,6 +299,34 @@ const prepostAppointmentsFactory = ({ elite2Api, appointmentsService, existingEv
         ...prepostAppointments,
       })
 
+      const userEmailData = await oauthApi.userEmail(res.locals, username)
+
+      const templateId = '391bb0e0-89b3-4aef-b11e-c6550b71fee8'
```

Comment:
Is it worth moving this into config.js?

Reply:
I can't decide yet. I am kind of hoping we end up with something abstracted when we settle on how and when we send emails to people. Left it almost on purpose to look ugly. Happy to move to config though, easily done.